governance/votes/Executive vote - May 14, 2021.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Template - [Executive Vote] Parameter Changes and Oracle Updates - May 14, 2021
+
+---
+# [Executive Proposal] Parameter Changes and Oracle Updates - May 14, 2021
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+### Parameter Changes
+
+Based on the recommendations of the [Open Market Committee](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-4-may-2021/7835) the [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee) and [Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling/) parameters will be updated for the vault types below.
+- The Stability Fee for KNC-A will be increased to 5.0%, TUSD-A and PAXUSD-A will be increased to 1.0%.
```

Comment:
Would be good to have these on separate lines and in the from x% to y% format. Maybe TUSD and PAX on a new line together?